packages/nextjs/components/Navbar.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Image from "next/image";
+import { SwitchTheme } from "./SwitchTheme";
+
+const Navbar = () => {
+  return (
+    <nav className="navbar p-2 bg-base-100 max-w-screen-2xl mx-auto flex justify-between items-center ">
+      <div className="flex gap-2 items-center">
+        <div className="max-w-[24px]">
+          <Image src="/assets/svg/logo.svg" width={24} height={24} className="w-full" alt="logo" />
+        </div>
+        <h3 className="text-op-neutral-900  font-bold text-sm sm:text-xl  m-0">Retro Funding</h3>
```

Comment:
Instead of text here, lets use this SVG that covers both the OP logo and the "Retro Funding" text. This way we don't to deal with the font differences.
```
<svg width="168" height="24" viewBox="0 0 168 24" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M160.793 23.994C160.247 23.994 159.687 23.973 159.113 23.931C158.553 23.889 158.028 23.833 157.538 23.763V21.327C158.042 21.383 158.581 21.432 159.155 21.474C159.729 21.53 160.268 21.558 160.772 21.558C161.71 21.558 162.466 21.446 163.04 21.222C163.628 20.998 164.055 20.641 164.321 20.151C164.601 19.675 164.741 19.066 164.741 18.324V16.077L165.371 14.649C165.329 15.699 165.112 16.595 164.72 17.337C164.328 18.065 163.796 18.625 163.124 19.017C162.452 19.395 161.675 19.584 160.793 19.584C160.009 19.584 159.295 19.437 158.651 19.143C158.021 18.849 157.475 18.443 157.013 17.925C156.565 17.407 156.215 16.798 155.963 16.098C155.711 15.398 155.585 14.649 155.585 13.851V13.41C155.585 12.612 155.711 11.87 155.963 11.184C156.229 10.484 156.6 9.87499 157.076 9.35699C157.552 8.83899 158.119 8.43999 158.777 8.15999C159.435 7.87999 160.163 7.73999 160.961 7.73999C161.899 7.73999 162.711 7.94299 163.397 8.34899C164.097 8.75499 164.643 9.34299 165.035 10.113C165.427 10.883 165.644 11.828 165.686 12.948L165.182 13.032V8.09699H167.492V18.156C167.492 19.542 167.254 20.662 166.778 21.516C166.302 22.37 165.567 22.993 164.573 23.385C163.593 23.791 162.333 23.994 160.793 23.994ZM161.633 17.085C162.193 17.085 162.704 16.959 163.166 16.707C163.628 16.455 163.999 16.091 164.279 15.615C164.559 15.139 164.699 14.565 164.699 13.893V13.158C164.699 12.514 164.552 11.968 164.258 11.52C163.978 11.072 163.607 10.736 163.145 10.512C162.683 10.274 162.179 10.155 161.633 10.155C161.017 10.155 160.471 10.302 159.995 10.596C159.533 10.876 159.169 11.275 158.903 11.793C158.637 12.311 158.504 12.927 158.504 13.641C158.504 14.341 158.637 14.95 158.903 15.468C159.169 15.986 159.533 16.385 159.995 16.665C160.471 16.945 161.017 17.085 161.633 17.085Z" fill="#05060B"/>
<path d="M143.773 19.5V8.09699H146.083V12.99H145.873C145.873 11.828 146.027 10.862 146.335 10.092C146.643 9.30799 147.098 8.71999 147.7 8.32799C148.316 7.93599 149.079 7.73999 149.989 7.73999H150.115C151.473 7.73999 152.502 8.18099 153.202 9.06299C153.902 9.93099 154.252 11.233 154.252 12.969V19.5H151.333V12.717C151.333 12.017 151.13 11.45 150.724 11.016C150.332 10.582 149.786 10.365 149.086 10.365C148.372 10.365 147.791 10.589 147.343 11.037C146.909 11.471 146.692 12.059 146.692 12.801V19.5H143.773Z" fill="#05060B"/>
<path d="M138.54 19.5V8.09703H141.459V19.5H138.54ZM136.944 10.281V8.09703H141.459V10.281H136.944ZM139.611 6.73204C139.037 6.73204 138.61 6.58504 138.33 6.29104C138.064 5.98304 137.931 5.59804 137.931 5.13604C137.931 4.67404 138.064 4.29604 138.33 4.00204C138.61 3.70804 139.037 3.56104 139.611 3.56104C140.185 3.56104 140.605 3.70804 140.871 4.00204C141.137 4.29604 141.27 4.67404 141.27 5.13604C141.27 5.59804 141.137 5.98304 140.871 6.29104C140.605 6.58504 140.185 6.73204 139.611 6.73204Z" fill="#05060B"/>
<path d="M129.086 19.878C128.288 19.878 127.553 19.731 126.881 19.437C126.209 19.143 125.628 18.73 125.138 18.198C124.648 17.666 124.27 17.05 124.004 16.35C123.738 15.636 123.605 14.866 123.605 14.04V13.599C123.605 12.787 123.731 12.024 123.983 11.31C124.249 10.596 124.613 9.97304 125.075 9.44104C125.551 8.90904 126.118 8.49604 126.776 8.20204C127.448 7.89404 128.19 7.74004 129.002 7.74004C129.898 7.74004 130.682 7.93604 131.354 8.32804C132.04 8.70604 132.586 9.28004 132.992 10.05C133.398 10.82 133.622 11.786 133.664 12.948L132.803 11.94V4.17004H135.722V19.5H133.412V14.649H133.916C133.874 15.811 133.636 16.784 133.202 17.568C132.768 18.338 132.194 18.919 131.48 19.311C130.78 19.689 129.982 19.878 129.086 19.878ZM129.737 17.421C130.311 17.421 130.836 17.295 131.312 17.043C131.788 16.777 132.166 16.399 132.446 15.909C132.74 15.405 132.887 14.817 132.887 14.145V13.305C132.887 12.633 132.74 12.073 132.446 11.625C132.152 11.163 131.767 10.813 131.291 10.575C130.815 10.323 130.297 10.197 129.737 10.197C129.107 10.197 128.547 10.351 128.057 10.659C127.581 10.953 127.203 11.373 126.923 11.919C126.657 12.465 126.524 13.102 126.524 13.83C126.524 14.558 126.664 15.195 126.944 15.741C127.224 16.273 127.602 16.686 128.078 16.98C128.568 17.274 129.121 17.421 129.737 17.421Z" fill="#05060B"/>
<path d="M111.793 19.5V8.09699H114.103V12.99H113.893C113.893 11.828 114.047 10.862 114.355 10.092C114.663 9.30799 115.118 8.71999 115.72 8.32799C116.336 7.93599 117.099 7.73999 118.009 7.73999H118.135C119.493 7.73999 120.522 8.18099 121.222 9.06299C121.922 9.93099 122.272 11.233 122.272 12.969V19.5H119.353V12.717C119.353 12.017 119.15 11.45 118.744 11.016C118.352 10.582 117.806 10.365 117.106 10.365C116.392 10.365 115.811 10.589 115.363 11.037C114.929 11.471 114.712 12.059 114.712 12.801V19.5H111.793Z" fill="#05060B"/>
<path d="M103.316 19.857C102 19.857 100.978 19.423 100.25 18.555C99.5364 17.687 99.1794 16.399 99.1794 14.6911V8.07605H102.098V14.943C102.098 15.643 102.294 16.2031 102.686 16.6231C103.078 17.0291 103.61 17.232 104.282 17.232C104.954 17.232 105.5 17.0151 105.92 16.5811C106.354 16.1471 106.571 15.5591 106.571 14.8171V8.07605H109.49V19.5001H107.18V14.6491H107.411C107.411 15.797 107.264 16.7561 106.97 17.5261C106.676 18.2961 106.235 18.8771 105.647 19.2691C105.059 19.6611 104.324 19.857 103.442 19.857H103.316Z" fill="#05060B"/>
<path d="M89.1255 19.5V4.17004H92.1075V19.5H89.1255ZM91.6875 13.116V10.596L97.8195 10.575V13.095L91.6875 13.116ZM91.6875 6.69004V4.17004H98.1135V6.69004H91.6875Z" fill="#05060B"/>
<path d="M77.6904 19.899C76.6824 19.899 75.7934 19.738 75.0234 19.416C74.2534 19.094 73.6024 18.66 73.0704 18.114C72.5384 17.554 72.1324 16.924 71.8524 16.224C71.5864 15.524 71.4534 14.796 71.4534 14.04V13.599C71.4534 12.815 71.5934 12.073 71.8734 11.373C72.1674 10.659 72.5804 10.029 73.1124 9.483C73.6584 8.923 74.3164 8.489 75.0864 8.181C75.8564 7.859 76.7244 7.698 77.6904 7.698C78.6564 7.698 79.5244 7.859 80.2944 8.181C81.0644 8.489 81.7154 8.923 82.2474 9.483C82.7934 10.029 83.2064 10.659 83.4864 11.373C83.7664 12.073 83.9064 12.815 83.9064 13.599V14.04C83.9064 14.796 83.7664 15.524 83.4864 16.224C83.2204 16.924 82.8214 17.554 82.2894 18.114C81.7574 18.66 81.1064 19.094 80.3364 19.416C79.5664 19.738 78.6844 19.899 77.6904 19.899ZM77.6904 17.4C78.4044 17.4 79.0064 17.246 79.4964 16.938C79.9864 16.616 80.3574 16.189 80.6094 15.657C80.8614 15.111 80.9874 14.495 80.9874 13.809C80.9874 13.109 80.8544 12.493 80.5884 11.961C80.3364 11.415 79.9584 10.988 79.4544 10.68C78.9644 10.358 78.3764 10.197 77.6904 10.197C77.0044 10.197 76.4094 10.358 75.9054 10.68C75.4154 10.988 75.0374 11.415 74.7714 11.961C74.5054 12.493 74.3724 13.109 74.3724 13.809C74.3724 14.495 74.4984 15.111 74.7504 15.657C75.0164 16.189 75.3944 16.616 75.8844 16.938C76.3744 17.246 76.9764 17.4 77.6904 17.4Z" fill="#05060B"/>
<path d="M64.5845 19.5001V8.09707H66.8945V12.9271H66.8315C66.8315 11.2891 67.1815 10.0501 67.8815 9.21007C68.5815 8.37007 69.6105 7.95007 70.9685 7.95007H71.3885V10.4911H70.5905C69.6105 10.4911 68.8475 10.7571 68.3015 11.2891C67.7695 11.8071 67.5035 12.5631 67.5035 13.5571V19.5001H64.5845Z" fill="#05060B"/>
<path d="M61.5466 19.647C60.4966 19.647 59.6286 19.514 58.9426 19.248C58.2706 18.968 57.7666 18.506 57.4306 17.862C57.0946 17.204 56.9266 16.322 56.9266 15.216L56.9476 4.88403H59.6776L59.6566 15.405C59.6566 15.965 59.8036 16.399 60.0976 16.707C60.4056 17.001 60.8396 17.148 61.3996 17.148H63.1846V19.647H61.5466ZM55.1206 10.239V8.09703H63.1846V10.239H55.1206Z" fill="#05060B"/>
<path d="M49.7211 19.899C48.7411 19.899 47.8801 19.731 47.1381 19.395C46.4101 19.059 45.8011 18.611 45.3111 18.051C44.8351 17.477 44.4711 16.84 44.2191 16.14C43.9811 15.44 43.8621 14.726 43.8621 13.998V13.599C43.8621 12.843 43.9811 12.115 44.2191 11.415C44.4711 10.701 44.8351 10.071 45.3111 9.525C45.8011 8.965 46.4031 8.524 47.1171 8.202C47.8311 7.866 48.6571 7.698 49.5951 7.698C50.8271 7.698 51.8561 7.971 52.6821 8.517C53.5221 9.049 54.1521 9.756 54.5721 10.638C54.9921 11.506 55.2021 12.444 55.2021 13.452V14.502H45.1011V12.717H53.3751L52.4721 13.599C52.4721 12.871 52.3671 12.248 52.1571 11.73C51.9471 11.212 51.6251 10.813 51.1911 10.533C50.7711 10.253 50.2391 10.113 49.5951 10.113C48.9511 10.113 48.4051 10.26 47.9571 10.554C47.5091 10.848 47.1661 11.275 46.9281 11.835C46.7041 12.381 46.5921 13.039 46.5921 13.809C46.5921 14.523 46.7041 15.16 46.9281 15.72C47.1521 16.266 47.4951 16.7 47.9571 17.022C48.4191 17.33 49.0071 17.484 49.7211 17.484C50.4351 17.484 51.0161 17.344 51.4641 17.064C51.9121 16.77 52.1991 16.413 52.3251 15.993H55.0131C54.8451 16.777 54.5231 17.463 54.0471 18.051C53.5711 18.639 52.9621 19.094 52.2201 19.416C51.4921 19.738 50.6591 19.899 49.7211 19.899Z" fill="#05060B"/>
<path d="M30.99 19.5001V4.06506H33.972V19.5001H30.99ZM40.356 19.5001L35.652 12.8221H38.991L43.842 19.5001H40.356ZM33.09 14.3761V11.9611H36.996C37.542 11.9611 38.011 11.8491 38.403 11.6251C38.809 11.4011 39.124 11.0861 39.348 10.6801C39.572 10.2741 39.684 9.81206 39.684 9.29406C39.684 8.76206 39.572 8.29306 39.348 7.88706C39.124 7.48106 38.809 7.16606 38.403 6.94206C38.011 6.71806 37.542 6.60606 36.996 6.60606H33.09V4.06506H36.681C37.913 4.06506 38.977 4.24706 39.873 4.61106C40.783 4.97506 41.483 5.52806 41.973 6.27006C42.463 7.01206 42.708 7.94306 42.708 9.06306V9.39906C42.708 10.5331 42.456 11.4641 41.952 12.1921C41.462 12.9201 40.769 13.4661 39.873 13.8301C38.977 14.1941 37.913 14.3761 36.681 14.3761H33.09Z" fill="#05060B"/>
<g clip-path="url(#clip0_218_192)">
<circle cx="12" cy="12" r="12" fill="#FF0420"/>
<path d="M8.50235 15.1894C7.78782 15.1894 7.20239 15.0212 6.74605 14.685C6.29572 14.3427 6.07056 13.8564 6.07056 13.2259C6.07056 13.0938 6.08557 12.9317 6.11559 12.7396C6.19365 12.3072 6.30473 11.7879 6.44883 11.1814C6.85714 9.5302 7.91091 8.70459 9.61016 8.70459C10.0725 8.70459 10.4868 8.78265 10.8531 8.93876C11.2193 9.08887 11.5076 9.31704 11.7177 9.62327C11.9279 9.92349 12.0329 10.2838 12.0329 10.7041C12.0329 10.8302 12.0179 10.9893 11.9879 11.1814C11.8978 11.7158 11.7898 12.2352 11.6637 12.7396C11.4535 13.5622 11.0902 14.1776 10.5739 14.5859C10.0575 14.9882 9.36698 15.1894 8.50235 15.1894ZM8.62844 13.8924C8.96469 13.8924 9.24989 13.7933 9.48407 13.5952C9.72424 13.397 9.89537 13.0938 9.99744 12.6855C10.1355 12.1211 10.2406 11.6287 10.3127 11.2084C10.3367 11.0823 10.3487 10.9532 10.3487 10.8211C10.3487 10.2747 10.0635 10.0015 9.49307 10.0015C9.15683 10.0015 8.86861 10.1006 8.62844 10.2988C8.39427 10.4969 8.22614 10.8001 8.12407 11.2084C8.01599 11.6107 7.90791 12.1031 7.79983 12.6855C7.77581 12.8056 7.7638 12.9317 7.7638 13.0638C7.7638 13.6162 8.05201 13.8924 8.62844 13.8924Z" fill="white"/>
<path d="M12.4465 15.0993C12.3804 15.0993 12.3294 15.0783 12.2934 15.0362C12.2633 14.9882 12.2543 14.9342 12.2663 14.8741L13.5093 9.01982C13.5213 8.95377 13.5543 8.89973 13.6083 8.8577C13.6624 8.81567 13.7194 8.79466 13.7795 8.79466H16.1752C16.8417 8.79466 17.3761 8.93276 17.7784 9.20896C18.1867 9.48516 18.3908 9.88446 18.3908 10.4068C18.3908 10.557 18.3728 10.7131 18.3368 10.8752C18.1867 11.5657 17.8835 12.0761 17.4271 12.4063C16.9768 12.7366 16.3583 12.9017 15.5718 12.9017H14.3559L13.9416 14.8741C13.9296 14.9402 13.8965 14.9942 13.8425 15.0362C13.7885 15.0783 13.7314 15.0993 13.6714 15.0993H12.4465ZM15.6348 11.6588C15.887 11.6588 16.1062 11.5897 16.2923 11.4516C16.4844 11.3135 16.6105 11.1154 16.6706 10.8572C16.6886 10.7551 16.6976 10.665 16.6976 10.587C16.6976 10.4128 16.6466 10.2807 16.5445 10.1907C16.4424 10.0946 16.2683 10.0466 16.0221 10.0466H14.9413L14.5991 11.6588H15.6348Z" fill="white"/>
</g>
<defs>
<clipPath id="clip0_218_192">
<rect width="24" height="24" fill="white"/>
</clipPath>
</defs>
</svg>
```

Reply:
when its on dark theme, the text will not be white